Dockerfile

```diff
@@ -8,19 +8,15 @@ WORKDIR /src
 COPY go.mod .
 COPY go.sum .
 
-RUN go mod download
-
 COPY . .
 
-RUN make build
```

Comment:
`/src/bin/` would be empty without `make build`?

Reply:
Sorry, I forget to recover them after debugging.